src/apps/vpn/translations/extras/languages.json

```diff
@@ -78,32 +78,7 @@
       "XXX": "¤"
     },
     "languages": {
-      "bg": "Lingua bulgara",
```

Comment:
This part is suspicious... I wonder why so many lines deleted.

Reply:
Agreed. Reverted these changes.